tasks/sass.js

```diff
@@ -6,6 +6,11 @@ const componentsFolderName = config.global.components.slice(1);
 const resourcesFolderName = config.global.resources.slice(1);
 const scssResourcesFolderName = path.join(resourcesFolderName, 'scss');
 
+const ignoreConfig = config.sass.excludePaths.map(
```

Comment:
Q: are `config` and `excludePaths` extendable through the `projectConfig.js` file?

Reply:
you can add it like this to your projectConfig.js - do you mean that?
```
sass: {excludePaths: [
		'components/**/*.scss'
	]},
```